### PR TITLE
Update the router-sidecar README.md file

### PR DESCRIPTION
**Pull request by uma-kt**
- Rephrased the sentences
- Update the hyperlinks
- Add the link to download the docker image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,11 +5,11 @@
 * The Slice Operator manages the lifecycle of a Slice Router by overseeing the deployment, configuration, continuous monitoring, and management of the Slice Router.
 * The Slice Router provides a full mesh network connectivity between the application pods and slice gateway pods in a cluster. 
 
-## Getting Started 
+## Get Started 
 
 Please refer to our documentation on:
-- [Installing KubeSlice on cloud clusters](https://kubeslice.io/documentation/open-source/0.5.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller).
-- [Installing KubeSlice on kind clusters](https://kubeslice.io/documentation/open-source/0.5.0/tutorials/kind-install-kubeslice-controller).
+- [Install KubeSlice on cloud clusters](https://kubeslice.io/documentation/open-source/0.5.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller).
```
Comment (veenamj-avesha) [inline README.md:11]: change the versions to 0.6.0 (the latest)

File: README.md [hunk 2]
```diff
@@ -47,48 +47,48 @@ helm repo update
...
-```
+   ```bash
+   git clone https://github.com/kubeslice/router-sidecar.git
+   cd router-sidecar
+   ```
 
-2. Adjust the `VERSION` variable in the Makefile to change the docker tag to be built.
-   Image is set as `docker.io/aveshasystems/router-sidecar:$(VERSION)` in the Makefile. Change this if required.
+2. Edit the `VERSION` variable in the Makefile to change the docker tag to be built.
+   Image is set as `docker.io/aveshasystems/router-sidecar:$(VERSION)` in the Makefile. Modify this if required.
```
Comment (veenamj-avesha) [inline README.md:56]: The image is set ....

File: README.md (hunk 1 above)
Comment (uma-kt) [reply README.md:11]: done

File: README.md (hunk 2 above)
Comment (uma-kt) [reply README.md:56]: done

**Review by veenamj-avesha — APPROVED** [review]
Changes LGTM

**Pull request merged**